key-features.md

```diff
@@ -26,7 +26,7 @@ We believe that being able to replicate in both directions lowers the risk when
 
 TiDB internally shards table into small range-based chunks that we refer to as "Regions". Each Region defaults to approximately 100MiB in size, and TiDB uses a Two-phase commit internally to ensure that Regions are maintained in a transactionally consistent way.
 
-Transactions in TiDB are strongly consistent, with snapshot isolation level consistency. For more information, see transaction [behavior and performance differences](/reference/transactions/transaction-model.md). This makes TiDB more comparable to traditional relational databases in semantics than some of the newer NoSQL systems using eventual consistency.
+Transactions in TiDB are strongly consistent, with snapshot isolation level consistency. For more information, see transaction [behavior and performance differences](/reference/transactions/transaction-isolation.md). This makes TiDB more comparable to traditional relational databases in semantics than some of the newer NoSQL systems using eventual consistency.
```

Comment:
This change is irrelevant to the Chinese version. 
Generally, I will replace `transaction-model.md`(now deleted) with `transaction-optimistic.md`. But here, according to the context, I replace it with `transaction-isolation.md`.